otter/models/cass.py

```diff
@@ -1486,6 +1510,46 @@ def _do_webhook_lookup(webhook_rec):
         d.addCallback(_do_webhook_lookup)
         return d
 
+    def get_webhook_index_only(self):
+        """
+        Get webhook info that is there in webhook index but is not there in
+        webhook_keys table
+
+        :return: Effect of CQLQueryExecute
+        """
+        query = ('SELECT "tenantId", "groupId", "policyId", "webhookKey" '
+                 'FROM {cf}')
+        eff = parallel(
+            [Effect(
+                CQLQueryExecute(query=query.format(cf=self.webhooks_table),
+                                params={},
+                                consistency_level=ConsistencyLevel.ONE)),
+             Effect(
+                CQLQueryExecute(query=query.format(cf=self.webhook_keys_table),
+                                params={},
+                                consistency_level=ConsistencyLevel.ONE))])
+        return eff.on(
+            lambda (whooks, wkeys): set(freeze(whooks)) - set(freeze(wkeys)))
+
+    def add_webhook_keys(self, webhook_keys):
+        """
+        Add webhook keys to webhook_keys table
+
+        :return: Effect of CQLQueryExecute
```

Comment:
The "Effect of *" idiom should generally be used to describe the *result* of the Effect, not the intent of the effect. So, in this case it'd be an Effect of... whatever connection.execute() will return in this case. (None?)